setup.py

```diff
@@ -18,7 +18,12 @@
       'License :: Public Domain',
       'Programming Language :: Python',
       'Topic :: Software Development :: Debuggers',
-      ]
-    )
+    ],
+    entry_points={
+        'console_scripts': [
+            'flamegraph = flamegraph.__main__:main',
```

Comment:
**Note to reviewers**: this allows running from the command-line as `flamegraph` (in addition to the existing `python -m flamegraph`).